BraveWallet/WalletStrings.swift

```diff
@@ -1940,5 +1940,33 @@ extension Strings {
       value: "Sent",
       comment: "As in sent cryptocurrency from one asset to another"
     )
+    public static let settingsEnableBiometricsTitle = NSLocalizedString(
+      "wallet.settingsEnableBiometricsTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Allow Biometric Unlock",
+      comment: "A label beside the toggle for allowing biometrics to unlock the wallet in wallet settings."
+    )
+    public static let settingsEnableBiometricsFooter = NSLocalizedString(
+      "wallet.settingsEnableBiometricsFooter",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Toggle on if you'd like to use biometrics (Face ID / Touch ID) to unlock your wallet",
+      comment: "The footer beneath the toggle for allowing biometrics to unlock the wallet in wallet settings."
+    )
+    public static let enterPasswordForBiometricsNavTitle = NSLocalizedString(
+      "wallet.enterPasswordForBiometricsTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enable Biometrics",
+      comment: "The navigation bar title displayed on the screen to enter wallet password to enable biometrics unlock from wallet settings."
+    )
+    public static let enterPasswordForBiometricsTitle = NSLocalizedString(
+      "wallet.enterPasswordForBiometricsTitle",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Enter passcode to enable biometrics for wallet (you'll only be asked for this once)",
```

Comment:
passcode here should probably be "your wallet password" or something, otherwise it almost seems like you should be entering your phone passcode